wpilibNewCommands/src/test/native/cpp/frc2/command/CommandDecoratorTest.cpp

```diff
@@ -335,6 +335,39 @@ TEST_F(CommandDecoratorTest, RaceWithOrder) {
   EXPECT_TRUE(firstWasPolled);
 }
 
+TEST_F(CommandDecoratorTest, Repeatedly) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly();
+
+  scheduler.Schedule(command);
+
+  for (int i = 1; i <= 50; i++) {
+    scheduler.Run();
+    EXPECT_EQ(i, counter);
+  }
+
+  EXPECT_TRUE(scheduler.IsScheduled(command));
+}
+
+TEST_F(CommandDecoratorTest, RepeatFor) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly(3);
+
+  scheduler.Schedule(command);
+  for (int i = 0; scheduler.IsScheduled(command); i++) {
+    scheduler.Run();
+    EXPECT_EQ(i + 1, counter);
+  }
+
+  EXPECT_EQ(3, counter);
```

Comment:
```suggestion
  EXPECT_TRUE(scheduler.IsScheduled(command));
  for (int i = 1; i < 3; i++) {
    scheduler.Run();
    EXPECT_EQ(i, counter);
    EXPECT_TRUE(scheduler.IsScheduled(command));
  }
  
  EXPECT_EQ(3, counter);
  EXPECT_FALSE(scheduler.IsScheduled(command));
```
See Java. I'm not as familiar with GTest logging, so it'd be worth experimenting locally to see whether a failure of the isScheduled check inside the loop will include the value of i.

Reply:
same as the Java one this is missing a `scheduler.Run()` on line 408 before the checks making it fail, so I will add that and then commit this.

Reply:
committed, see 6e88f89